src/CoreBundle/DataAccess/MenuItemDataAccess.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\DataAccess;
+
+use ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar\MenuCategory;
+use ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar\MenuItem;
+use ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar\MenuItemFactoryInterface;
+use ChameleonSystem\CoreBundle\DataModel\MenuCategoryAndItem;
+
+class MenuItemDataAccess implements MenuItemDataAccessInterface
+{
+    /**
+     * @var MenuItemFactoryInterface
+     */
+    private $menuItemFactory;
```

Comment:
```suggestion
    private MenuItemFactoryInterface $menuItemFactory;
```

and remove the docblock

Reply:
Is changed.